src/map/ai/states/magic_state.cpp

```diff
@@ -665,6 +661,11 @@ void CMagicState::FinishSpell()
 
         action.messageID = msg;
 
+        if (PTarget->objtype == TYPE_MOB && msg != 31)
```

Comment:
hello magic number, you seem to be missing an enum or a comment defining what you are supposed to be